renderdoc/driver/gl/gl_manager.cpp

```diff
@@ -84,6 +84,9 @@ void GLResourceManager::MarkFBOAttachmentsReferenced(ResourceId fboid, GLResourc
 {
   FBOCache *cache = m_FBOAttachmentsCache[fboid];
 
+  if(!record)
```

Comment:
What is the purpose of this change? This doesn't have anything to do with platform-specific support as far as I can tell. Record should always be valid going into this function.

Reply:
It is quickfix for program that somehow made record nullptr, I forgot to remove it